ui/app/AppLayouts/Chat/ChatColumn/ChatComponents/SignTransactionModal.qml

```diff
@@ -31,10 +31,6 @@ ModalPopup {
         root.close()
     }
 
-    function estimateGas(){
-        gasSelector.estimateGas()
-    }
-
```

Comment:
This function is never used anywhere in the current and child components, nor the parent